scripts/install-tree-sitter-lib

```diff
@@ -52,9 +52,18 @@ dir_name=$(dirname "$BASH_SOURCE")
 "$dir_name"/update-version-symlinks
 "$dir_name"/download-tree-sitter --lazy
 
+# For the Makefile of tree-sitter
+if [[ -z "${CC:-}" ]]; then
+  CC=$(opam var sys-ocaml-cc 2>/dev/null || \
+         opam exec -- ocamlc -config-var c_compiler 2>/dev/null || \
+         opam exec -- ocamlopt -config-var c_compiler 2>/dev/null)
+fi
+: ${AR:=${CC:+$(echo $CC | sed -E 's/(-?)[^-]*$/\1ar/')}}
+: ${STRIP:=${CC:+$(echo $CC | sed -E 's/(-?)[^-]*$/\1strip/')}}
+
 (
   cd downloads/tree-sitter
-  make PREFIX="$prefix"
+  make PREFIX="$prefix" "${CC:+CC=$CC}" "${AR:+AR=$AR}" "${STRIP:+STRIP=$STRIP}"
```

Comment:
This change isn't needed if we export `CC`, `AR`, and `STRIP` as environment variables, I'm not sure what's the cleanliest option.

Reply:
```
PREFIX="$prefix" "${CC:+CC=$CC}" "${AR:+AR=$AR}" "${STRIP:+STRIP=$STRIP}" make
```
would rely on Bash to set the environment variables just for the `make` invocation, instead of relying on `make` to do something similar.

Either way makes it clear what command needs these variables.

Reply:
> Edit: no, `"${CC:+CC=$CC}"` before make doesn't work because it doesn't start literally with `CC=`. Nevermind.

Nice, I didn't know. I'll leave it explicit as-is, then.